Inc/TASRun.h

```diff
@@ -9,10 +9,10 @@
 
 #define MAX_SIZE 1024
 #define MAX_CONTROLLERS 2
-#define MAX_DATA_LANES 3
+#define MAX_DATA_LANES 4

```

Comment:
Why is this 4? D0, D1, and D2 leaves a max of 3 lanes per controller -- unless I am mistaken

Reply:
4 lanes are actually for two reasons.

Firstly SNES multitap support - although SNES has only 2 data lanes, I'm using lanes 3 and 4 as the extra virtual multitap lanes that are sent as lanes 1 and 2 when the SEL line goes low.

Secondly for making USB playback easier - having 4 lanes means that .r16m frames can be passed directly into the ExtractDataAndAddFrame function without doing any byte shifting.